test/fake/plugin_server.go

```diff
@@ -35,14 +35,11 @@ func NewPluginServer(cfg PluginConfig) (string, func() error) {
 
 	basePath := fmt.Sprintf("%s:%d", cfg.Server.Host, cfg.Server.Port)
 	builder := plugin.NewIndexBuilder(loggerx.NewNoop())
-
+	idx, err := builder.Build(cfg.BinariesDirectory, basePath+"/static", ".*")
```

Comment:
I put that inside the handler as I wanted to rebuild it each time botkube calls the endpoint. I'm using:
```
env PLUGIN_SERVER_HOST=http://host.k3d.internal go run test/helpers/plugin_server.go
```

as a long living process. So I don't need to restart that if I rebuild the binary, as it will fail on checksum. Can we revert that?